src/drf_yasg/utils.py

```diff
@@ -287,7 +288,13 @@ def make_schema_definition():
         # swagger 2.0 does not support specifics about file fields, so ImageFile gets no special treatment
         # OpenAPI 3.0 does support it, so a future implementation could handle this better
         err = SwaggerGenerationError("parameter of type file is supported only in a formData Parameter")
-        if swagger_object_type != openapi.Parameter:
+        if swagger_object_type == openapi.Schema:
+            # FileField.to_representation returns URL or file name
+            if getattr(field, 'use_url', api_settings.UPLOADED_FILES_USE_URL):
+                return SwaggerType(type=openapi.TYPE_STRING, format=openapi.FORMAT_URI)
```

Comment:
Since this only applies to response schemas, would it make sense to always return a `read_only` Schema here?

Reply:
I just realized that doing that would involve moving some code around (from [here](https://github.com/axnsan12/drf-yasg/blob/a2c21539f783480e99177c64b0fb79bd61f3e318/src/drf_yasg/utils.py#L216) to [here](https://github.com/axnsan12/drf-yasg/blob/a2c21539f783480e99177c64b0fb79bd61f3e318/src/drf_yasg/utils.py#L179)) and adding some checks to prevent the value on the field overriding the value passed as argument; if you don't to deal with that I could handle it.